includes/device-groups.inc.php

```diff
@@ -66,7 +66,7 @@ function GenGroupSQL($pattern, $search='') {
         $search .= ' &&';
     }
 
-    $sql = 'SELECT DISTINCT('.str_replace('(', '', $tables[0]).'.device_id) FROM '.implode(',', $tables).' WHERE '.$search.' ('.str_replace(array('%', '@', '!~', '~'), array('', '.*', 'NOT REGEXP', 'REGEXP'), $pattern).')';
+    $sql = 'SELECT DISTINCT('.str_replace('(', '', $tables[0]).'.device_id),`devices`.* FROM '.implode(',', $tables).' WHERE '.$search.' ('.str_replace(array('%', '@', '!~', '~'), array('', '.*', 'NOT REGEXP', 'REGEXP'), $pattern).')';
```

Comment:
Yeah you're right.

It's used twice with GetGroupsFromDevice(). I've created another function that mimics that one as I needed the full response back. If I update the use of GetGroupsFromDevice() to use the full output from dbFetchRows instead are you ok with that?